### ISSUE TITLE
examples include json parse fail

### ISSUE THREAD

**Issue by sigoden** [raml-org/raml-js-parser-2#485]
```
examples:
    a: !include examples/a.json
    b: !include examples/b.json
```
```
TypeError: Cannot read property 'isEmpty' of undefined
    at  /node_modules/raml-1-parser/dist/raml1/wrapped-ast/wrapperHelper.js:677:83
```
the problem code is:
```
var expandables = runtimeDefinition.examples().filter(function (x) { return !x.isEmpty() && x.isSingle() == isSingle; });
```
After some debug, i found the function `runtimeDefinition.examples()`  return `[undefined, undefined] `

when I repalce include statement with the value object, it work.  when examples facet have property with an include statement, it throw the error

**Comment by ddenisenko** [comment]
Reproducible. Thank you for reporting.

**Comment by eduramiba** [comment]
This also makes the parser fail:

```raml
examples:
  simplest: 
    value: !include examples/accounts/get_200.json
    description: JSON output example for the simplest call GET /accounts
    displayName: Basic view JSON output example
```

But this works fine for me:

```raml
examples:
  simplest: !include examples/accounts/get_200.json
```

**Comment by KonstantinSviridov** [comment]
Thhank you, guys, for reporting. The fix will be released in 1.1.5
